artifacts/openapi-spec/public.yaml

```diff
@@ -0,0 +1,562 @@
+      operationId: put-models-modelId
+    delete:
+      summary: Delete model
+      tags:
+        - models
+      responses:
+        '200':
+          description: OK
+      operationId: delete-models-modelId
+  '/models/{modelId}/data':
+    parameters:
+      - schema:
+          type: string
+        name: modelId
+        in: path
+        required: true
+    post:
+      summary: Add data
+      tags:
+        - models
+      responses:
+        '200':
+          description: OK
+      operationId: post-models-modelId-data
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                content:
+                  type: string
+    get:
+      summary: List data
+      operationId: get-models-modelId-data
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Data_info'
+      tags:
+        - models
+  '/models/{modelId}/data/{dataId}':
+    parameters:
+      - schema:
+          type: string
+        name: modelId
+        in: path
+        required: true
+      - schema:
+          type: string
+        name: dataId
+        in: path
+        required: true
+    get:
+      summary: Get data
+      tags:
+        - models
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Data_info'
+      operationId: get-models-modelId-data-dataId
+    put:
+      summary: Update data
+      tags:
+        - models
+      responses:
+        '200':
+          description: OK
+      operationId: put-models-modelId-data-dataId
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                content:
+                  type: string
+      description: ''
+    delete:
+      summary: Delete data
+      tags:
+        - models
+      responses:
+        '200':
+          description: OK
+      operationId: delete-models-modelId-data-dataId
+  /solvers:
+    get:
+      summary: List solvers
+      tags:
+        - solvers
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+              examples:
+                example-1:
+                  value:
+                    - string
+      operationId: get-solvers
+    put:
+      summary: Update solvers
+      operationId: put-solvers
+      responses:
+        '200':
+          description: OK
+      tags:
+        - solvers
```

Comment:
Kan man tilføje en attribut med navnet på den solver som skal tilføjes, evt. navnet på et Docker image som indeholder solveren?

Det kan også være det skal undersøges separat, hvordan vi lige laver dette?

Reply:
ja, mangler en eller anden form for identifier